parsl/config.py

```diff
@@ -64,6 +66,7 @@ def __init__(self,
                  retries: int = 0,
                  run_dir: str = 'runinfo',
                  strategy: Optional[str] = 'simple',
+                 max_idletime: Optional[float] = 120.0,
```

Comment:
This type should be `float`, not `Optional[float]`.

`Optional[float]` means "this value may be `None` or `float`" - but `None` does not work with the simple strategy.

This is different from using the word "optional" to mean "there is a default float value so you do not need to explicitly specify one" - for example, look how the `usage_tracking` parameter is not `Optional[bool]`.

Reply:
Good point. Fixed this.